apps/redi-connect/src/pages/app/me/Me.tsx

```diff
@@ -31,18 +34,35 @@ import {
 import { REDI_LOCATION_NAMES } from '@talent-connect/shared-config'
 import { useLoading } from '../../../hooks/WithLoading'
 // CHECK OUT THE LOADER
+import { useHistory } from 'react-router-dom'
 import './Me.scss'
 import OnboardingSteps from './OnboardingSteps'
 
 function Me() {
   const queryClient = useQueryClient()
   const submitProfileForReviewMutation = useConProfileSubmitForReviewMutation()
   const { Loading, isLoading } = useLoading()
+  const history = useHistory()
+
   const myProfileResult = useLoadMyProfileQuery(
     {
       loopbackUserId: getAccessTokenFromLocalStorage().userId,
     },
-    { onSuccess: () => console.log('Me loaded it') }
+    {
+      onSuccess: (profile) => {
+        console.log('Me loaded it')
```

Comment:
@helloanil, I see this error logged in the console before I see a successful log-in:
![image](https://github.com/talent-connect/connect/assets/51786805/a2cb0fd5-a3b5-4343-973b-63566fd8efb1)

Can we fix it?